cluster/images/etcd/migrate-if-needed.sh

```diff
@@ -227,7 +262,7 @@ for step in ${SUPPORTED_VERSIONS}; do
       echo "Starting etcd ${step} in v3 mode failed"
       exit 1
     fi
-    ${ETCDCTL_CMD} rm --recursive "${ETCD_DATA_PREFIX}"
+    ${ETCDCTL_CMD} --endpoints "http://127.0.0.1:${ETCD_PORT}" rm --recursive "${ETCD_DATA_PREFIX}"
```

Comment:
@wojtek-t FYI. We found this when adding tests. It looks like this wasn't getting run correctly (maybe ever?) since the `--endpoints` flag was not being set to the etcd ports `etcd_start` starts etcd on.

Reply:
Makes sense. But let's do this in a separate PR.